src/utils/utils.cpp

```diff
@@ -192,48 +206,115 @@ void PrintValue(std::ostream &os, const mg_path *path) {
   }
 }
 
-void PrintValue(std::ostream &os, const mg_value *value) {
-  switch (mg_value_get_type(value)) {
-    case MG_VALUE_TYPE_NULL:
-      os << "Null";
-      return;
-    case MG_VALUE_TYPE_BOOL:
-      os << (mg_value_bool(value) ? "true" : "false");
-      return;
-    case MG_VALUE_TYPE_INTEGER:
-      os << mg_value_integer(value);
-      return;
-    case MG_VALUE_TYPE_FLOAT:
-      os << mg_value_float(value);
-      return;
-    case MG_VALUE_TYPE_STRING:
-      PrintValue(os, mg_value_string(value));
-      return;
-    case MG_VALUE_TYPE_LIST:
-      PrintValue(os, mg_value_list(value));
-      return;
-    case MG_VALUE_TYPE_MAP:
-      PrintValue(os, mg_value_map(value));
-      return;
-    case MG_VALUE_TYPE_NODE:
-      PrintValue(os, mg_value_node(value));
-      return;
-    case MG_VALUE_TYPE_RELATIONSHIP:
-      PrintValue(os, mg_value_relationship(value));
-      return;
-    case MG_VALUE_TYPE_UNBOUND_RELATIONSHIP:
-      PrintValue(os, mg_value_unbound_relationship(value));
-      return;
-    case MG_VALUE_TYPE_PATH:
-      PrintValue(os, mg_value_path(value));
-      return;
-    default:
-      os << "{unknown value}";
-      break;
-  }
+void PrintValue(std::ostream &os, const mg_date *date) {
+  const auto days = date::days(mg_date_days(date));
+  const auto sys_days = date::sys_days(days);
+  const auto ymd = date::year_month_day(sys_days);
+  os << ymd;
+}
+
+void PrintValue(std::ostream &os, const mg_local_time *local_time) {
+  const auto nanos =
+      std::chrono::nanoseconds(mg_local_time_nanoseconds(local_time));
```

Comment:
consider namespace aliases to make this shorter and fit it into one liners:
`namespace std::chrono = chrono;`